### ISSUE TITLE
bug using features

### ISSUE THREAD

**Issue by neok-m4700** [conda/conda-build#2122]
**environ.py** around l. 645:
```python
def get_install_actions(...)
    # ...
    for feature, value in feature_list:
        if value:
            specs.append('%s@' % feature)
```
is throwing a `AttributeError: 'tuple' object has no attribute 'append'` on specs.

a possible correction could be https://github.com/neok-m4700/conda-build/commit/863f3c0bf4379a678bc5a219d71a10a6769a35d2

**Comment by msarahan** [comment]
Seems like the right fix.  Can you propose a test case?  Is there a recipe you'd recommend as a test case?

**Comment by neok-m4700** [comment]
I've played around with the features https://github.com/neok-m4700/test_features.git

It works ie setting the correct env variable picks the correct feature, is it sufficient ?

**Comment by msarahan** [comment]
fixed by #2428, closing.
